src/bson/bson-config.h

```diff
@@ -174,6 +148,18 @@
 #endif
 
 
+/*
+ * Define to 1 if you have arc4random_buf available on your platform.
+ */
+#define BSON_HAVE_ARC4RANDOM_BUF 0
```

Comment:
Where did BSON_HAVE_ARC4RANDOM_BUF come from? I don't see it at all in the C driver source.

Reply:
For this and the others please refer to this file: https://github.com/mongodb/mongo-c-driver/blob/master/src/libbson/src/bson/bson-config.h.in
Specifically, the section at the bottom:
```
/*
 * Define to 1 if you have arc4random_buf available on your platform.
 */
#define BSON_HAVE_ARC4RANDOM_BUF @BSON_HAVE_ARC4RANDOM_BUF@
#if BSON_HAVE_ARC4RANDOM_BUF != 1
# undef BSON_HAVE_ARC4RANDOM_BUF
#endif


/*
 * Define to 1 if you have rand_r available on your platform.
 */
#define BSON_HAVE_RAND_R @BSON_HAVE_RAND_R@
#if BSON_HAVE_RAND_R != 1
# undef BSON_HAVE_RAND_R
#endif


/*
 * Define to 1 if you have strlcpy available on your platform.
 */
#define BSON_HAVE_STRLCPY @BSON_HAVE_STRLCPY@
#if BSON_HAVE_STRLCPY != 1
# undef BSON_HAVE_STRLCPY
#endif
```

Reply:
Yes but BSON_HAVE_ARC4RANDOM_BUF doesn't exist in 1.20.0: https://github.com/mongodb/mongo-c-driver/blob/1.20.0/src/libbson/src/bson/bson-config.h.in

So I'm wondering how this ended up here.